packages/gestalt-codemods/18.0.0/__testfixtures__/scrollableContainer-rename-scrollBoundaryContainer.input.js

```diff
@@ -0,0 +1,11 @@
+// @flow strict
+import React from 'react';
+import { ScrollableContainer, Box } from 'gestalt';
```

Comment:
@AlbertCarreras might be good to also add a test for aliased imports:
```
import { ScrollableContainer as CustomContainer, Box } from 'gestalt';
```

Reply:
DONE